tox.ini

```diff
@@ -15,7 +15,7 @@ deps =
     -rrequirements.txt
     -rdev-requirements.txt
 commands =
-    wget -N https://oss.sonatype.org/content/repositories/releases/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar
+    wget -N {env:SWAGGER_URL:https://oss.sonatype.org/content/repositories/releases/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar}
     java -jar swagger-codegen-cli-2.3.1.jar generate -i openapis/swagger.yaml -l python -o swagger_client
     bash -ec "cd swagger_client/; python setup.py install; cd .."
-    py.test {posargs: --cov=indexd -lvv tests/}
+    py.test {posargs: --cov=indexd -lv tests/ --cov-report=html --cov-report=xml --junit-xml=test-reports/results.xml}
```

Comment:
Can you add `--cov-report term` so it also shows up on the terminal

Reply:
Is this something we want for all services? Jenkins is already producing the html and xml reports not sure how much term would add in addition? I am no expert with tox so if you say this should be in I can go around and add it to the different services 👍

Reply:
Doesn't add any extra overhead, it just provides a summary on the console so one can at least see that, before trying to look at the generated html. It will be nice if we have it for all the projects, right now just a few projects do it.

Reply:
All right, I have updated this PR to include coverage report in terminal